validations_libs/constants.py

```diff
@@ -0,0 +1,26 @@
+#   Copyright 2020 Red Hat, Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+#
+
+import os
+
+DEFAULT_VALIDATIONS_BASEDIR = '/usr/share/validations-common'
+
+ANSIBLE_VALIDATION_DIR = '/usr/share/validations-common/playbooks'
+
+VALIDATION_GROUPS_INFO = '%s/groups.yaml' % DEFAULT_VALIDATIONS_BASEDIR
+
+VALIDATION_GROUPS = ['no-op',
+                     'prep',
+                     'post']
```

Comment:
needs to retrieve those groups from the groups.yaml file (see https://review.opendev.org/#/c/704521/)